src/api/v1beta1/properties_test.go

```diff
@@ -117,8 +117,20 @@ func TestTokens(t *testing.T) {
 
 func TestTenantUUID(t *testing.T) {
 	t.Run("happy path", func(t *testing.T) {
-		apiUrl := "https://demo.dev.dynatracelabs.com/api"
-		expectedTenantId := "demo"
+		const apiUrl = "https://demo.dev.dynatracelabs.com/api"
```

Comment:
why change this to `const` in all these tests?

Reply:
For consistency with the new test case I added. Constants should be preferred when possible.

Reply:
I don't think adding `const` to every variable in a test case is good idea
We've only had file/package scoped constants which can be reused for now, and I don't think we should introduce them on a function level

Reply:
If this was a function unrelated to anything (was a util function used by different parts of the codebase) then it would make sense

AFAIK however this function is only used by `func (dk *DynaKube) TenantUUID() (string, error)` and is package private, so adding `const` feels like adding noise to the code more then anything else IMO

Reply:
but if we want to be super-duper sure that it doesn't change anything, then its fine 😄

Reply:
Matter of taste. I can revert the change if that's what you prefer, no big deal.